jobs/slack-statistics.ts

```diff
@@ -18,16 +18,187 @@ export async function postStatisticsToSlack() {
+    const tutorKnowsCoronaSchoolFromTable: [string, string][] = [];
+    for (let i = 0; i < tutorKnowsCoronaSchoolFromValue.length; i++) {
+        tutorKnowsCoronaSchoolFromTable[i] = [tutorKnowsCoronaSchoolFromValue[i], tutorKnowsCoronaSchoolFromCount[i]];
+    }
+
+    // Instructor screening - total
+    const instructorScreeningCount = await prisma.instructor_screening.count({
+        where: {
+            createdAt: { gte: begin, lte: end },
+        },
+    });
+
+    // Instructor screening - success
+    const instructorScreeningSuccessCount = await prisma.instructor_screening.count({
+        where: {
+            createdAt: { gte: begin, lte: end },
+            success: true,
+        },
+    });
+
+    // Grouping 'knowsCoronaSchoolFrom' - Instructor
+    const instructorKnowsCoronaSchoolFrom = await prisma.instructor_screening.groupBy({
+        by: ['knowsCoronaSchoolFrom'],
+        where: {
+            createdAt: { gte: begin, lte: end },
+            knowsCoronaSchoolFrom: { notIn: [''] },
+        },
+        _count: {
+            createdAt: true,
+        },
+    });
+
+    const instructorKnowsCoronaSchoolFromValue = instructorKnowsCoronaSchoolFrom.map(function (item) {
+        return item['knowsCoronaSchoolFrom'];
+    });
+
+    const instructorKnowsCoronaSchoolFromCount = instructorKnowsCoronaSchoolFrom.map(function (item) {
+        return String(item['_count']['createdAt']);
+    });
+
+    const instructorKnowsCoronaSchoolFromTable: [string, string][] = [];
+    for (let i = 0; i < instructorKnowsCoronaSchoolFromValue.length; i++) {
+        instructorKnowsCoronaSchoolFromTable[i] = [instructorKnowsCoronaSchoolFromValue[i], instructorKnowsCoronaSchoolFromCount[i]];
+    }
+
+    // Matches
+    const matchCount = await prisma.match.count({
+        where: {
+            createdAt: { gte: begin, lte: end },
+        },
+    });
+
+    // New subcourses created
+    const subcourseCreatedCount = await prisma.subcourse.count({
+        where: {
+            createdAt: { gte: begin, lte: end },
+        },
+    });
+
+    // Number of current subcourses
+    const subcourseCurrentCount = await prisma.subcourse.count();
+
+    // TODO: number of free / taken seats of current public subcourses
+    const subcourseSeats = await prisma.course.findMany({
+        select: {
+            id: true,
+            name: true,
+            subcourse: {
+                select: {
+                    maxParticipants: true,
+                    // eslint-disable-next-line camelcase
+                    subcourse_participants_pupil: {
+                        select: {
+                            pupilId: true,
+                        },
+                    },
+                },
+            },
+        },
+    });
+
+    const subcourseSeatsCount = subcourseSeats.map((course) => {
+        const subcourses = course.subcourse;
+        const totalSeats = subcourses.reduce((acc, subcourse) => acc + subcourse.maxParticipants, 0);
```

Comment:
(if we query by subcourse this would be obsolete. Also semantically this does not make that much sense, as subcourses do have participants, courses are just "templates" that multiple instructors could use for their subcourse)